src/js/common/utils/labels.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
I don't think we should copy this whole file. Let's make a labels file with just the ones we need in the `common/schemaform` folder.